### PR TITLE
DW-21 working pledge

### PR DESCRIPTION
**Pull request by lizasautter**

### CODE REVIEW

File: src/components/ActiveDares.js [hunk 1]
```diff
@@ -5,26 +5,51 @@ import ReactPlayer from 'react-player'
...
+import PledgeButton from './pledge'
 
 import '../css/userprofile.css'
 
 
 class ActiveDares extends Component {
+  state = {
+    totalAmount: 0
+  }
+
```
Comment (nblange) [inline src/components/ActiveDares.js]: get rid of commented code

File: src/components/ActiveDares.js [hunk 2]
```diff
@@ -5,26 +5,51 @@ import ReactPlayer from 'react-player'
...
+  // }
 
   completeDareButton = () => {
     this.props.viewDare(this.props.dare);
     this.props.changePageMode('CompleteDare');
   }
 
   personalAccount () {
-    console.log('tessssssttttttttt', this.props.id)
+    console.log('tessssssttttttttt', this.props)
```
Comment (nblange) [inline src/components/ActiveDares.js]: do we want this console.log still

**Review by nblange — COMMENTED** [review]
corine is going to look at the completed dare.js just review comments

File: src/components/CompletedDares.js [hunk 3]
```diff
@@ -27,17 +27,19 @@ class CompletedDares extends Component {
     })
   }
 
-  // renderCompletedDares(dare) {
-  //   var self = this;
-  //   return (
-  //     <CompletedDaresComp changePageMode={self.props.changePageMode} dare={dare} key={dare.id} />
-  //   )
-  // }
```
Comment (CorinneEarl) [inline src/components/CompletedDares.js:35]: we need to keep dare in place of element for completed dares

File: src/components/CompletedDares.js [hunk 4]
```diff
@@ -27,17 +27,19 @@ class CompletedDares extends Component {
...
+    return (
+      <CompletedDaresComp changePageMode={self.props.changePageMode} element={element} key={i} />
+    )
+  }
+
 
   render() {
     return (
       <div className="TilesContainer">
-      {this.state.compdareBlocks.map((dare, i) => (<CompletedDaresComp changePageMode={this.props.changePageMode} dare={dare} key={i} />))}
```
Comment (CorinneEarl) [inline src/components/CompletedDares.js:40]: same as above

**Review by CorinneEarl — APPROVED** [review]

**Review by lizasautter — COMMENTED** [review]
Done

**Pull request merged**